doc/index.md

```diff
@@ -456,6 +456,19 @@ Returns: a Promise that is fulled one the web socket is connected
   and other necessary async tasks are complete.
 
 
+#### `startSoftphoneSession()`
+Starts a softphone call session with the given peer or peers.
+
+`initialize()` must be called first.
+
+Declaration:
+``` ts
+startSoftphoneSession(softphoneSessionParams: IStartSoftphoneSessionParams): Promise<any>;
+```
+Params:
+* `softphoneSessionParams: IStartSoftphoneSessionParams` Required: Contains participant information for placing the call. See [softphone#IStartSoftphoneSessionParams](softphone.md#istartsoftphonesessionparams) for full details on the request parameters.
+
+Returns: a promise with an object containing the session details.
```

Comment:
NIT (but not blocking), maybe add the type you created in the function (`{id: string, selfUri: string}`) here and line 466

Reply:
Good catch on line 466. This has been updated.